msteams-platform/task-modules-and-cards/cards/Universal-actions-for-adaptive-cards/Work-with-universal-actions-for-adaptive-cards.md

```diff
@@ -0,0 +1,105 @@
+---
+title: Work with universal actions for adaptive cards
+description: Work with the universal actions for adaptive cards.
+ms.topic: conceptual
+---
+
+# Work with universal actions for adaptive cards
+
+A universal actions for adaptive cards provides a way to generalize bots and the Bot Framework to implement adaptive card-based scenarios for both Teams and Outlook. This document covers the following:
+
+* [Refresh model](#refresh-model)
+* [`adaptiveCard/action` invoke activity](#adaptivecardaction-invoke-activity)
+* [Schema used for universal actions for adaptive cards](#schema-for-universal-actions-for-adaptive-cards)
+* [Backward compatibility](#backward-compatibility)
+
+## Schema for universal actions for adaptive cards
+
+The universal actions for adaptive cards is introduced in the adaptive cards schema version 1.4. To use the adaptive card effectively, the `version` property of your adaptive card must be set to 1.4 or higher.
+
+> [!NOTE]
+> Setting the `version` property to 1.4 makes your adaptive card incompatible with older clients of the platforms or applications, such as Outlook and Teams, as they do not support the universal actions for adaptive cards.
+
+If you set the card version to less than 1.4 and use either or both, `refresh` property and `Action.Execute`, the following happens:
+
+| Client | Behavior |
+| :-- | :-- |
+| Outlook | Your card stops working. Card is not refreshed and `Action.Execute` does not render or your card is rejected. |
+| Teams | Your card stops working. Card is not refreshed and `Action.Execute` does not render depending on the version of the Teams client. To ensure maximum compatibility in Teams, define `Action.Execute` with an `Action.Submit` in the fallback property. |
+
+For more information on how to support older clients, see [backward compatibility](#backward-compatibility).
+
+### Action.Execute
+
+When authoring adaptive cards, replace `Action.Submit` and `Action.Http` with `Action.Execute`. The schema for `Action.Execute` is similar to that of `Action.Submit`. 
+
+[Action.Execute schema and properties](https://docs.microsoft.com/en-us/adaptive-cards/authoring-cards/universal-action-model#actionexecute)
+
+Now you can use the refresh model to allow adaptive cards to update automatically.
+
+## Refresh model
+
+To automatically refresh your adaptive card, define its `refresh` property, which embeds an action of type `Action.Execute` and an `userIds` array. 
+
+[Refresh schema and properties](https://docs.microsoft.com/en-us/adaptive-cards/authoring-cards/universal-action-model#refresh-mechanism)
+
+[How to get user MRI's in the conversation to add in userIds list in refresh section of adaptive card?](https://docs.microsoft.com/en-us/microsoftteams/platform/bots/how-to/get-teams-context?tabs=dotnet#fetch-the-roster-or-user-profile)
```

Comment:
What do you think about creating a subsection on userIds where we combine all the notes and points on user ids including the content you have in FAQs here?

Reply:
Great suggestion. Done!